src/app/models/propTypes/inline/index.js

```diff
@@ -5,7 +5,6 @@ import { inlineLinkBlockPropTypes } from '../inlineLink';
 
 export const inlineModelPropTypes = {
   language: string.isRequired,
-  text: string.isRequired,
```

Comment:
Dont we want to keep this? Now that your passing it in anyway?

Reply:
Your passing text on https://github.com/bbc/simorgh/pull/3306/files#diff-1b925d2c9ac672c84c04aeeae7166d46R38 to satisfy the proptype, while also removing the prop type?

Reply:
Yes.